roaring/roaring.go

```diff
@@ -73,7 +73,7 @@ type Containers interface {
 
 	// PutContainerValues updates an existing container at key.
 	// If a container does not exist for key, a new one is allocated.
-	PutContainerValues(key uint64, containerType byte, n int, mapped bool)
+	PutContainerValues(key uint64, containerType byte, n int32, mapped bool)
```

Comment:
ah, we technically can't do this - changes the API in an exported interface.

Reply:
man , why you creating tickets we can't do :)

Reply:
to be fair, when I created the ticket, I don't think this interface even existed. Of course, we can still make the change on the `Container` struct which is the important part.

Reply:
I'll make the adjustment, but it does now add a risk of overflow that would normally be protected by the compiler